vcpkg-configuration.json

```
@@ -28,7 +28,7 @@
         "zlib",
         "zstd"
       ],
-      "baseline": "e36e5c537beddb430e78dc1ed91b95729032086e"
+      "baseline": "e9bfff323eeb94290fd59f26c8550c374d0bba85"
```

```suggestion
      "baseline": "8ad4537e4074510927208ebc3102b2c3ddbf4ac2"
```
I believe this was the commit from the branch before the merge. The new commit should be the one currently at the head of master.